src/System.Management.Automation/namespaces/FileSystemContentStream.cs

```diff
@@ -843,7 +843,11 @@ private void CreateStreams(string filePath, string streamName, FileMode fileMode
             {
                 if (!String.IsNullOrEmpty(streamName))
                 {
+#if !UNIX
                     _stream = AlternateDataStreamUtilities.CreateFileStream(filePath, streamName, fileMode, fileAccess, fileShare);
+#else
+                    throw new PlatformNotSupportedException();
```

Comment:
I think it's better to make sure `CreateStreams` is never used for an alternate stream instead of inserting PlatformNotSupportedException.